### PR TITLE
feat(module:badge): support size

### PR DESCRIPTION
**Pull request by buqiyuan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: components/badge/badge-sup.component.ts [hunk 1]
```diff
@@ -67,6 +68,7 @@ export class NzBadgeSupComponent implements OnInit, OnChanges {
   @Input() disableAnimation = false;
   @Input() nzCount?: number | TemplateRef<NzSafeAny>;
   @Input() noAnimation = false;
+  @Input() nzSize: 'default' | 'small' = 'default';
```
Comment (simplejason) [inline components/badge/badge-sup.component.ts:71]: Please add some tests for `nzSize`, and use `NzSizeDSType` instead of string :)

**Review by simplejason — CHANGES_REQUESTED** [review]

File: components/badge/badge-sup.component.ts (hunk 1 above)
Comment (buqiyuan) [reply components/badge/badge-sup.component.ts:71]: Ok, I will correct it in a few days

Comment (buqiyuan) [reply components/badge/badge-sup.component.ts]: done

**Comment by simplejason** [comment]
👍 点赞，如果你有钉钉账号的话并且愿意的话，我希望将你拉到钉钉群和微信群的开发者群中 :)

**Comment by simplejason** [comment]
> buqiyuan

钉钉号有吗？按名称搜不到用户，或者可以从zorro群加我哈

**Comment by buqiyuan** [comment]
> > buqiyuan
> 
> 钉钉号有吗？按名称搜不到用户，或者可以从zorro群加我哈

image
文档中分享的群二维码貌似过期了诶，我个人钉钉号确实是： `buqiyuan`，怪了  搜不到吗

**Comment by simplejason** [comment]
> > > buqiyuan
> > 
> > 
> > 钉钉号有吗？按名称搜不到用户，或者可以从zorro群加我哈
> 
> image 文档中分享的群二维码貌似过期了

ng-zorro-dingtalk

**Comment by simplejason** [comment]
这个是你吗
image

**Comment by buqiyuan** [comment]
> 这个是你吗 image

是的

**Pull request merged**